cmd/doracled/cmd/register_oracle.go

```diff
@@ -60,13 +67,11 @@ func registerOracleCmd() *cobra.Command {
 			// sign and broadcast to Panacea
 			msgRegisterOracle := oracletypes.NewMsgRegisterOracle(uniqueID, oracleAccount.GetAddress(), nodePubKey, nodePubKeyRemoteReport, trustedBlockInfo.TrustedBlockHeight, trustedBlockInfo.TrustedBlockHash)
 
-			cli, txBuilder, err := generateGrpcClientAndTxBuilder(conf)
+			txBuilder := panacea.NewTxBuilder(*queryClient)
+			cli, err := panacea.NewGrpcClient(conf)
```

Comment:
You may need to close the gRPC connection using defer func.
When you see the `panacea.NerGrpcClient()`, it creates a connection but doesn't close the connection.
So I think caller should close the connection.

Reply:
Thanks, I add defer function in [f7b488e](https://github.com/medibloc/panacea-doracle/pull/46/commits/f7b488e1dd94f4dfc16eac4e9b5036b9ba09d41c).